[OKTA-1147649] Token exchange updates for renaming (#6120)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/ai-agent-token-exchange/main/index.md

#### What you need

- An Okta org that's subscribed to Okta for AI Agents.
- An Okta user account with the super admin role.
- [Add and register AI agents](https://help.okta.com/okta_help.htm?type=oie&id=ai-agent-register) in your Okta org.

> - **Resource Connections** is configured for the AI agent, defining which resources it's allowed to access. See [AI agent resource connections](https://help.okta.com/okta_help.htm?type=oie&id=ai-agent-secure).

Reviewer: ```suggestion
- **Resource connections** are configured for the AI agent, defining which resources it's allowed to access. See [AI agent resource connections](https://help.okta.com/okta_help.htm?type=oie&id=ai-agent-secure).
```

- An OIDC web app is configured to authenticate users and obtain an ID token.

---

## Overview

> You've [registered an AI agent](https://help.okta.com/okta_help.htm?type=oie&id=ai-agent-register). You have also defined its access to third-party resources integrated with your Okta org using **Resource Connections**. Now, the agent must obtain the actual tokens or credentials to perform tasks.

Reviewer: ```suggestion
You've [registered an AI agent](https://help.okta.com/okta_help.htm?type=oie&id=ai-agent-register). You've also created resource connections that define the AI agent's access to your org's resources. Now, the agent must obtain the actual tokens or credentials to perform tasks.
```
